Libplanet/Blocks/BlockHeader.cs

```diff
@@ -5,24 +5,47 @@
 
 namespace Libplanet.Blocks
 {
-    internal readonly struct BlockHeader
+    /// <summary>
+    /// Block header containing information about <see cref="Block{T}"/>s except transactions.
+    /// </summary>
+    public readonly struct BlockHeader
     {
-        public static readonly byte[] IndexKey = { 0x69 }; // 'i'
-
-        public static readonly byte[] TimestampKey = { 0x74 }; // 't'
-
-        public static readonly byte[] DifficultyKey = { 0x64 }; // 'd'
-
-        public static readonly byte[] NonceKey = { 0x6e }; // 'n'
-
-        public static readonly byte[] MinerKey = { 0x6d }; // 'm'
-
-        public static readonly byte[] PreviousHashKey = { 0x70 }; // 'p'
-
-        public static readonly byte[] TxHashKey = { 0x78 }; // 'x'
-
-        public static readonly byte[] HashKey = { 0x68 }; // 'h'
-
+        private static readonly byte[] IndexKey = { 0x69 }; // 'i'
+
+        private static readonly byte[] TimestampKey = { 0x74 }; // 't'
+
+        private static readonly byte[] DifficultyKey = { 0x64 }; // 'd'
+
+        private static readonly byte[] NonceKey = { 0x6e }; // 'n'
+
+        private static readonly byte[] MinerKey = { 0x6d }; // 'm'
+
+        private static readonly byte[] PreviousHashKey = { 0x70 }; // 'p'
+
+        private static readonly byte[] TxHashKey = { 0x78 }; // 'x'
+
+        private static readonly byte[] HashKey = { 0x68 }; // 'h'
+
+        /// <summary>
+        /// Creates a <see cref="BlockHeader"/> instance.
+        /// </summary>
+        /// <param name="index">The height of the block to create.  Goes to the <see cref="Index"/>.
```

Comment:
```suggestion
        /// <param name="index">The height of the block.  Goes to the <see cref="Index"/>.
```
Block header is created after the block is mined, it's little awkward to say block to create.